File: mRemoteNG/Connection/Connection.Protocol.Rlogin.vb
Namespace Connection
    Namespace Protocol
        Public Class Rlogin
            Inherits Connection.Protocol.PuttyBase

            Public Sub New()
                Me.PuttyProtocol = Putty_Protocol.rlogin
            End Sub

            Public Enum Defaults
                Port = 513
            End Enum
        End Class
    End Namespace
End Namespace

File: mRemoteV1/Connection/Protocols/deprecated/Connection.Protocol.Rlogin.cs


namespace mRemoteNG.Connection.Protocol
{
	public class Rlogin : PuttyBase
	{
				
		public Rlogin()
		{
			this.PuttyProtocol = Putty_Protocol.rlogin;
		}
				
		public enum Defaults
		{
			Port = 513
		}
	}
}
